### ISSUE TITLE
Can't Recycle free FOB structures

### ISSUE THREAD

**Issue by JamieKG** [Wyqer/kp_liberation#36]
When trying to recycle a building

client RTP

```
18:58:48 Error in expression <elect 1) * GRLIB_recycling_percentage * _suppMulti);
_price_a = round ((_objecti>
18:58:48   Error position: <_suppMulti);
_price_a = round ((_objecti>
18:58:48   Error Undefined variable in expression: _suppmulti
18:58:48 File mpmissions\__CUR_MP.Tanoa\scripts\client\actions\do_recycle.sqf, line 44
18:58:48 Error in expression <") - 1) * -1);
_ammoMulti = _currentAmmo/_allAmmo;
_fuelMulti = fuel _vehtorecyc>
18:58:48   Error position: </_allAmmo;
_fuelMulti = fuel _vehtorecyc>
18:58:48   Error Zero divisor
18:58:48 File mpmissions\__CUR_MP.Tanoa\scripts\client\actions\do_recycle.sqf, line 21
18:58:50 Error in expression <nditionalSelect;

_supplyCrates = ceil (_price_s / 100);
_ammoCrates = ceil (_pr>
18:58:50   Error position: <_price_s / 100);
_ammoCrates = ceil (_pr>
18:58:50   Error Undefined variable in expression: _price_s
```

http://i.imgur.com/IrdwC1s.png

base part is not recycled 

Thanks JamieKG

**Comment by Wyqer** [comment]
Already fixed at be874c8dd0f9d378492746f87b814466caf65cd4
Was mentioned in #34

**Comment by JamieKG** [comment]
my bad im sure i updated
